flixel/addons/plugin/screengrab/FlxScreenGrab.hx

```diff
@@ -155,28 +152,54 @@ class FlxScreenGrab extends FlxBasic
 		{
 			Filename = Filename + ".png";
 		}
+		return Filename;
+	}
+	
+	private static function save(Filename:String = ""):Void
+	{
+		if (screenshot.bitmapData == null)
+		{
+			return;
+		}
+		
+		Filename = fixFilename(Filename);
+		
+		var png:ByteArray;
+	#if flash
+		png = PNGEncoder.encode(screenshot.bitmapData);
+	#elseif openfl_legacy
```

Comment:
Is there a difference between them? I think we use lime_legacy everywhere else, so that might make sense for consistency.